src/views/PatronsPreRegistrationListContainer/hooks/useCreateNewUser.js

```diff
@@ -0,0 +1,47 @@
+import { useMutation } from 'react-query';
+import { useHistory } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { FormattedMessage } from 'react-intl';
+
+import { useOkapiKy, useCallout } from '@folio/stripes/core';
+
+import { PATRON_PREREGISTRATIONS_API, PERMISSIONS_API } from '../../../constants';
+
+// this hook is used to create a new user from pre-registration record
+const useCreateNewUser = () => {
+  const callout = useCallout();
+  const history = useHistory();
+  const ky = useOkapiKy();
+
+  const {
+    isLoading,
+    mutateAsync: createUser,
+  } = useMutation({
+    mutationFn: async (user) => {
+      try {
+        const { userId } = await ky.put(`${PATRON_PREREGISTRATIONS_API}/${user.id}/mergeOrCreateUser`).json();
+        // create permission user
+        const permUserId = uuidv4();
```

Comment:
Doesn't BE handle it?

Reply:
BE will handle if UI is not providing it. I have removed it in my new commit.